consul/prepared_query_endpoint.go

```diff
@@ -41,7 +42,9 @@ func (p *PreparedQuery) Apply(args *structs.PreparedQueryRequest, reply *string)
 		// to collide since this isn't inside a write transaction.
 		state := p.srv.fsm.State()
 		for {
-			args.Query.ID = generateUUID()
+			if args.Query.ID, err = uuid.GenerateUUID(); err != nil {
```

Comment:
It's using the function's return parameter, `err`.  Upon entry into `Apply`, `err` is zero initialized and visible for the entire scope of `Apply`.